api/src/types/scopes.ts

```diff
@@ -24,6 +24,7 @@ enum Scope {
     CreateInterviews = 'create:interviews',
     DeleteInterviews = 'delete:interviews',
     ReadAllUsersRoles = 'read:all_users_roles',
+    DeleteUsersRole = 'delete:users_role',
```

Comment:
We also already have the `delete:roles` scope for admins

Reply:
Removed this line and replaced usage with the `delete:roles` scope

Reply:
Push changes?

Reply:
*face palm* lol pushed